dbld/rules

```diff
@@ -136,11 +136,6 @@ image: image-$(DEFAULT_IMAGE)
 image-%:
 	cd $(DBLD_DIR)/images/ && $(DOCKER) build --network=host -t balabit/syslog-ng-$* -f $*.dockerfile .
 
-pull-images: $(foreach image,$(BUILDER_IMAGES), pull-image-$(image))
-pull-image: pull-image-$(DEFAULT_IMAGE)
```

Comment:
These targets seemed useful. Is it necessary to remove them?

Maybe this is a rebase problem. For example, I miss setting --build-arg=OS_PLATFORM in this patch, on the other hand this change is not necessary.